### ISSUE TITLE
运行rt-detr时的问题

### ISSUE THREAD

**Issue by Kilig0215** [PaddlePaddle/Paddle#53135]
### 请提出你的问题 Please ask your question

运行的时候显示：
``W0420` 17:43:14.609467  9336 gpu_resources.cc:217] WARNING: device:  . The installed Paddle is compiled with CUDNN 8.4, but CUDNN version in your machine is 8.2, which may cause serious incompatible bug. Please recompile or reinstall Paddle with compatible CUDNN version.`
且只进行了1个epoch未报错就停止了
'[04/20 17:43:16] ppdet.engine INFO: Epoch: [0] [    0/29316] learning_rate: 0.000000 loss_class: 0.172974 loss_bbox: 2.508943 loss_giou: 1.847765 loss_class_aux: 1.054720 loss_bbox_aux: 15.031494 loss_giou_aux: 11.121633 loss_class_dn: 0.915602 loss_bbox_dn: 1.656143 loss_giou_dn: 1.335711 loss_class_aux_dn: 4.354774 loss_bbox_aux_dn: 8.280717 loss_giou_aux_dn: 6.678555 loss: 54.959030 eta: 48 days, 7:56:23 batch_cost: 1.9783 data_cost: 0.0000 ips: 2.0219 images/s'
我对此进行了测试：
`Python` 3.9.12 (main, Apr  4 2022, 05:22:27) [MSC v.1916 64 bit (AMD64)] :: Anaconda, Inc. on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import paddle
>>> paddle.version.cudnn()
'8.4.0'
>>>`
显示为8.4.0为什么会报warning呢

**Comment by lym0302** [comment]
您好，上面显示你的机器是 8.2，但是你安装的paddle 是 8.4 的，所以有兼容问题，你可以尝试重新安装相应的版本再试试

**Comment by Kilig0215** [comment]
> 您好，上面显示你的机器是 8.2，但是你安装的paddle 是 8.4 的，所以有兼容问题，你可以尝试重新安装相应的版本再试试



> 您好，上面显示你的机器是 8.2，但是你安装的paddle 是 8.4 的，所以有兼容问题，你可以尝试重新安装相应的版本再试试

谢谢您，我发现我将paddle.version.cudnn()这个函数理解为我存在的cudnn版本了

**Comment by Ligoml** [comment]
rt-detr模型放在PaddleDetection代码仓库：https://github.com/PaddlePaddle/PaddleDetection/tree/develop/configs/rtdetr ，使用问题可以去对应的repo下提issue哈
